polaris-tokens/src/colors-uplift.ts

```diff
@@ -0,0 +1,268 @@
+type ColorScale =
+  | '01'
```

Comment:
Agreed. JS has some interesting semantics here:

![Screenshot 2023-05-18 at 2 01 01 PM](https://github.com/Shopify/polaris/assets/32409546/a9ff653d-dba9-4c3d-b720-7f77d822ef12)

I'm not sure the `0` pad is adding value. It may even be confusing as devs will only be able to use "number" computed access with tokens above `10`.

Reply:
That makes sense. I think we can drop the leading 0. I was mainly thinking of it from a consumer side where we'd want the tokens in an easy to read order. So you don't see gray-1 and then gray-10, gray-11....gray-2 etc... but consumers shouldn't be using these directly anyway